plugin/src/main/java/io/spine/tools/gradle/bootstrap/JavaExtension.java

```diff
@@ -112,6 +116,40 @@ private void addSourceSets() {
         directoryStructure.register(sourceRoot);
     }
 
+    @Override
+    protected ImmutableMap<Dependency, String> forcedDependencies() {
+        return ImmutableMap.of(ForcedDependency.PROTOBUF_JAVA.dependency(),
+                               ForcedDependency.PROTOBUF_JAVA.version());
+    }
+
+    /**
+     * The dependencies whose particular versions are required in order for {@code JavaExtension}
+     * to work properly.
+     */
+    @VisibleForTesting
+    enum ForcedDependency {
+        PROTOBUF_JAVA("com.google.protobuf", "protobuf-java", "3.9.0");
```

Comment:
This will have to be updated each time the Protobuf version changes. Please consider using the existing ways of obtaining the Protobuf dependency spec.